packages/dev/materials/src/cell/cellMaterial.ts

```diff
@@ -20,6 +20,7 @@ import type { IAnimatable } from "core/Animations/animatable.interface";
 import "./cell.fragment";
 import "./cell.vertex";
 import { EffectFallbacks } from "core/Materials/effectFallbacks";
+import { addClipPlaneUniforms, bindClipPlane } from "core/Materials/clipPlaneMaterialHelper";

```

Comment:
Same.

Reply:
need core here as it is material library